frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -12,7 +13,19 @@ const HealthItemIcon: React.FC<HealthItemIconProps> = ({ state, dataTest }) => (
 );
 
 const HealthItem: React.FC<HealthItemProps> = React.memo(
-  ({ className, state, title, details, popupTitle, noIcon = false, icon, children }) => {
+  ({
+    className,
+    state,
+    title,
+    details,
+    popupTitle,
+    noIcon = false,
+    icon,
+    position,
+    hasAutoWidth,
+    maxWidth,
```

Comment:
```suggestion
    hasAutoWidth,
    maxWidth,
```
HealthItem is going to be a part of the Console SDK. So changes that allow users to deviate from the standard designs shouldn't be exposed from here. I would suggest we not make changes to the width.